backend/asmgen.ml

```diff
@@ -214,7 +214,11 @@ let rec regalloc ~ppf_dump round (fd : Mach.fundecl) =
     newfd
   end
 
-let count_duplicate_spills_reloads_in_block (cfg_with_infos : Cfg_with_infos.t) =
+let (++) x f = f x
+
+let id x = x
```

Comment:
Even though it is not widely used
(because fairly recent), maybe use
`Fun.id`.